retropie-shell-script-boilerplate.sh

```diff
@@ -1,33 +1,45 @@
 #!/usr/bin/env bash
 
-# [SCRIPT_TITLE] (e.g. Retropie Shell Script Boilerplate.)
+# [SCRIPT_NAME] (e.g.: script_template.sh)
 # [SCRIPT_DESCRIPTION] (e.g. A template for building shell scripts for RetroPie.)
 #
 # Author: [AUTHOR] (e.g. hiulit)
-# If you have the script in a GitHub Repository, use the comments below. If not, just remove them.
-# Repository: [REPOSITORY_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
-# Issues: [REPOSITORY_ISSUES_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate/issues)
+# Repository: [REPO_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
 # License: [LICENSE] [LICENSE_URL] (e.g. MIT https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate/blob/master/LICENSE)
 #
 # Requirements:
 # - RetroPie x.x.x (e.g. RetroPie 4.x.x)
-# - [NAME_OF_THE_PACKAGE] package (e.g. libav-tools package)
+# - [PACKAGE_NAME] (e.g. libav-tools)
 
-home="$(find /home -type d -name RetroPie -print -quit 2>/dev/null)"
```

Comment:
I think this version works better than
```
user="$SUDO_USER"
[[ -z "$user" ]] && user="$(id -un)"
home="$(eval echo ~$user)"
```

Remember that we had some issues in [RetroPie Fun Facts Splashscreens](https://github.com/hiulit/RetroPie-Fun-Facts-Splashscreens)? When trying to run the script in `/etc/rc.local` the user was always sudo and we wanted it to be pi...

Reply:
the fun-facts case was a very specific one, when the script **MUST** be run by root itself. And that  `find` trick is a workaround.

Maybe we can keep one method and let the other commented. It's up to you.

In my scripts I'll allways use the method without the `find` because on a system where there are many users, if other user has a directory named `RetroPie` it can bring unexpected results.

Reply:
Ok, then we'll use your method and then we can leave the other one commented saying something like "If you need to run the script in `/etc.rc.local` try to use this method instead of the other"?